Automattic/wp-calypso
# Framework: Rename .jsx files to .js

To play more nicely with http://2ality.com/2017/06/pkg-esnext.html, which I'm going to enable soon (for Project Hubris, and general tree-shaking goodness). Plus, the whole `.jsx` extension stuff is kinda obsolete anyway these days. (Edit: Okay, it's possible to use Webpack's `resolve.extensions` to work around this. Still obsolete.)

In a nutshell, untranspiled `.js` files are recognized by tools that can handle ES6(+) code (such as webpack), while untranspiled `.jsx` files aren't.

To test:
* `make clean && make run`. Takes a while.
* Test all the things! Specifically:
  * A Calypso section you know well
  * `/devdocs/design`
  * SSR: Check `/themes` and `/theme/mood` while logged out. Verify that the Page Source contains theme description etc.
